src/controllers/lacations.controller.js

```diff
@@ -27,8 +27,8 @@ class LocationsController {
     const locations = await LocationsService.getLocations();
     const data = [];
     locations.map(async (location) => {
-      const { country, city } = location;
-      data.push({ country, city });
+      const { country, city, id } = location;
+      data.push({ country, city, id });
```

Comment:
@Jajabenit250 Actually, you don't need to create another array and put data inside, array.map function return an array.